src/jsx.d.ts

```diff
@@ -622,6 +622,7 @@ export namespace JSXInternal {
 		alt?: string;
 		as?: string;
 		async?: boolean;
+		autoCapitalize?: string;
```

Comment:
According to the MDN link the `autocapitalize` value doesn't work with any string, but only with specific values. Something like `autocapitalize="asdasfa"` is invalid. The MDN page lists these values as valid ones: `off`, `none`, `on`, `sentences`, `words` and `characters`. Let's limit the typings to those values instead of allowing any string. We can do that via the union syntax:

```ts
autoCapitalize?: "off" | "none" | "on" | "sentences" | "words" | "characters";
```

Reply:
Thanks for the tip, hopefully this is still good one year later!